doc/api/net.md

```diff
@@ -416,109 +571,134 @@ See also: [`socket.setTimeout()`][]
 added: v0.1.90
 -->
 
-Returns the bound address, the address family name and port of the
-socket as reported by the operating system. Returns an object with
-three properties, e.g.
-`{ port: 12346, family: 'IPv4', address: '127.0.0.1' }`
+The `socket.address()` method returns the address to which the socket is
+bound, the address family name, and port of the socket as reported by the
+operating system.
+
+The object returned has the following properties:
+
+* `port` {number}
+* `family` {String} Either `'IPv4'` or `'IPv6'`
+* `address` {String} The IPv4 or IPv6 address
```

Comment:
unless its unbound, in which case the object has no properties. and wouldn't it be a path if it was a local socket?